docs/custom-flows/oauth-connections.mdx

```diff
@@ -118,60 +83,94 @@ Clerk ensures that security critical nonces will be passed only to allowlisted U
 
 So for maximum security in your production instances, you need to allowlist your custom redirect URLs via the [Clerk Dashboard](https://dashboard.clerk.com/) or the [Clerk Backend API](https://clerk.com/docs/reference/backend-api/tag/Redirect-URLs).
```

Comment:
I know you didn't change this sentence, but the link here doesn't seem to take you to any information about allowlisting a custom redirect URL, but maybe I missed it somewhere on the page.

Reply:
yes, its the create method. 
https://clerk.com/docs/reference/backend-api/tag/Redirect-URLs#operation/CreateRedirectURL

Reply:
I updated the copy to link to the backend sdk method instead 👍